composer.json

```diff
@@ -26,15 +33,35 @@
     },
     "sort-packages": true
   },
+  "extra": {
+    "enable-patching": true
+  },
   "autoload": {
     "psr-4": {
-      "Google\\Web_Stories\\": "includes"
-    }
+      "Google\\Web_Stories\\": "includes",
+      "AmpProject\\": "vendor/ampproject/amp-wp/lib/common/src",
+      "AmpProject\\Optimizer\\": "vendor/ampproject/amp-wp/lib/optimizer/src"
+    },
+    "exclude-from-files": [
+      "ampproject/amp-wp/back-compat/back-compat.php",
+      "ampproject/amp-wp/includes/amp-helper-functions.php",
+      "ampproject/amp-wp/includes/admin/functions.php",
+      "ampproject/amp-wp/includes/deprecated.php",
+      "ampproject/amp-wp/docs/includes/register-wp-cli-commands.php"
+    ]
   },
   "autoload-dev": {
     "psr-4": {
       "Google\\Web_Stories\\Tests\\": "tests/phpunit/includes"
     }
   },
+  "repositories": [
```

Comment:
Any chance we could use packagist here? It is much much faster.

Reply:
These are not yet on Packagist. Once all packages are refactored, split up and put on Packagist, we‘ll switch over.

Reply:
Should we document this? Add it to the to do?

Reply:
Added to the to-do above.

That should totally suffice, as we'll be working closely with the AMP plugin team on making this transition.